content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -21,8 +21,6 @@ To clarify the network traffic paths and make it easier to understand, this blog
 
 _Note 1: The following analysis is based on Istio 1.18.2, where ambient mode uses iptables for redirection._
 
-_Note 2: The communications between sidecar and ztunnel/waypoint proxy uses `[HTTP Based Overlay Network (HBONE)](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)`._
```

Comment:
Although I see we also have the link below.

Reply:
It was below, but I removed it because it was failing the spell check. _Note's closing _ was being read from the link by our linter. So because it was below, it was simpler to just remove.